lib/core/kernel.nit

```diff
@@ -1045,12 +1045,13 @@ universal Char
 	#
 	#     assert 'A'.is_whitespace  == false
 	#     assert ','.is_whitespace  == false
-	#     assert ' '.is_whitespace  == true
+	#     assert ' '.is_whitespace  == true # space
+	#     assert ' '.is_whitespace  == true # non-breaking space
```

Comment:
Considering the character itself is named "No-Break **SPACE**", that its category is, among others, **SPACE**, I figure considering it as a Space is the normal thing to do, not the other way around. I agree with this change